clang/docs/ReleaseNotes.rst

```diff
@@ -128,7 +128,10 @@ C2y Feature Support
   also removes UB with code like ``(void)(void)1;``.
 - Implemented `WG14 N3411 <https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3411.pdf>`_
   which allows a source file to not end with a newline character. This is still
-  reported as a conforming extension in earlier language modes.
+  reported as a conforming extension in earlier language modes. Note,
```

Comment:
I thought this is removing the 'reported as a conforming extension in earlier language modes'?  Or is my lack of caffeine showing?

Reply:
Nope, that's a think-o on my part. :-D